src/ui/components/DevTools.tsx

```diff
@@ -223,11 +223,11 @@ function _DevTools({
   }, [recording]);
 
   if (!loadingFinished) {
-    return <LoadingScreen fallbackMessage="Starting your session..." />;
+    return <LoadingScreen fallbackMessage="Loading..." />;
   }
 
   if (loadedRegions === null) {
```

Comment:
Perhaps merge these into one IF ` if (!loadingFinished || loadedRegions === null) {` Or better yet, can we move the  `loadedRegions === null` check into the `createSocket` call